libraries/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mylocaldep/libraries",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
Keep an eye on this one, it's recommended to build your library code to JS if you are creating public libraries (like I did in [`@use-expo/fonts`](https://github.com/byCedric/use-expo/blob/master/packages/font/package.json#L25-L32))